munit-cats-effect/shared/src/main/scala/munit/CatsEffectSuite.scala

```diff
@@ -0,0 +1,19 @@
+package munit
+
+import cats.effect.{ContextShift, IO, Timer}
+
+abstract class CatsEffectSuite extends FunSuite {
```

Comment:
👍

Reply:
I still have some open questions on this before I can continue :)